src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -115,19 +115,20 @@ public function syncPivot($model, $data)
 
                 // if a JSON was passed instead of an array, turn it into an array
                 if (is_string($values)) {
-                    $values = json_decode($values);
+                    $values = json_decode($values, true);
                 }
 
                 $relation_data = [];
-                foreach ($values as $pivot_id) {
-                    $pivot_data = [];
-
-                    if (isset($field['pivotFields'])) {
-                        foreach ($field['pivotFields'] as $pivot_field_name) {
-                            $pivot_data[$pivot_field_name] = $data[$pivot_field_name][$pivot_id];
-                        }
+                if (isset($field['fields'])) {
```

Comment:
Let's use `pivotFields` for now, in order to:
- differentiate them from `fields` in `repeatable` (they do different things, are saved differently)
- keep backwards-compatibility in naming (at least)

We might decide to change it back before this PR is merged, after we test with the interface. But for now let's use `pivotFields`.